charts/observation-service/templates/_helpers.tpl

```diff
@@ -0,0 +1,53 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "observation-service.name" -}}
```

Comment:
To make it consistent, Can i request you to add functions similar to this: https://github.com/caraml-dev/helm-charts/blob/main/charts/merlin/templates/_helpers.tpl#L8-L28?

Reply:
That said - do you want me to add service monitor too?

Reply:
That would be great if you can add it too. You can add a section for monitoring and if enabled create a service monitor too 🙏

Reply:
Added 🙏🏻

Reply:
I think you missed to use the newly added ones in`observation-service.name` and  `observation-service.fullname`. 

These helper functions are to be used for naming all the resources in the chart.

ref: https://github.com/caraml-dev/helm-charts/blob/main/charts/merlin/templates/_helpers.tpl#L31-L41